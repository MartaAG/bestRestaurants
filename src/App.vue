<template>
  <div id="app">
  <v-app id="inspire">
    <v-app id="inspire">


      <v-app-bar
        app
        color="indigo"
        dark
      >

        <v-toolbar-title>Best restaurants</v-toolbar-title>
      </v-app-bar>

      <v-main>
        <v-container
          class="fill-height"
          fluid
        >
          <v-row
            align="center"
            justify="center"
          >
            <v-col class="text-center">
              <List>

              </List>
              <v-tooltip left>
              
                <span>Source</span>
              </v-tooltip>
            </v-col>
          </v-row>
        </v-container>
      </v-main>
      <v-footer
        color="indigo"
        app
      >
        <span class="white--text">&copy; {{ new Date().getFullYear() }}</span>
      </v-footer>
    </v-app>
  </v-app>
</div>
</template>

<script>
import List from "./components/List";

export default {
  name: "App",

  components: {
    List
  },

  data: () => ({
    //
  })
};
</script>
