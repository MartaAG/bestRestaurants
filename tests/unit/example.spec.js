import Vuetify from 'vuetify'

import App from "@/App.vue";

import { mount, createLocalVue } from "@vue/test-utils";




describe("App.vue", () => {
  let vuetify;
  let wrapper;

  beforeEach(() => {
    const localVue = createLocalVue()
    vuetify = new Vuetify()
    localVue.use(vuetify)

    wrapper = mount(App, {
      localVue
    })
  })

  it('should match snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
  it('should render a vue instance', () => {
    expect(mount(App).isVueInstance()).toBe(true);
  })
  it('should have footer'), () => {
      expect(wrapper.contains("footer")).toBe(true);
  }
});
