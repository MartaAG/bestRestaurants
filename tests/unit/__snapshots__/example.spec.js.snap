// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.vue should match snapshot 1`] = `
<div id="app">
  <v-app id="inspire">
    <v-app id="inspire">
      <v-app-bar app="" color="brown darken-4" dark="">
        <v-toolbar-title>Best restaurants</v-toolbar-title>
      </v-app-bar>
      <v-main>
        <v-container class="fill-height">
          <v-row>
            <v-col cols="12">
              <router-view></router-view>
            </v-col>
          </v-row>
        </v-container>
      </v-main>
      <v-footer color="brown darken-4" app="" dark=""><span class="white--text">Â© 2020</span></v-footer>
    </v-app>
  </v-app>
</div>
`;
